client/pages/ReturnBike.tsx

```diff
@@ -109,6 +128,17 @@ const ReturnBike = ({navigation}) => {
             calculateTripTime();
         }
     }, [ride]);
+
```

Comment:
do we need to get permission again or can we store the status in AuthContext and then just update the location only?

Reply:
I slapped that in there just to make sure we never got stuck without location permissions. I'll see if I can lift this up so it only happens once. It might not even matter since it always asks when you land on the map for the first time.